subsidy/views.py

```diff
@@ -49,3 +52,20 @@ def subsidy_delete(request, subsidy_id):
     subsidy = get_object_or_404(Subsidy, id=subsidy_id)
     subsidy.delete()
     return redirect("/subsidy/list")
+
+def subsidy_update(request, subsidy_id):
+    subsidy= get_object_or_404(Subsidy, id=subsidy_id)
+
+    form= CreateNewSubsidy(instance=subsidy)
+    if request.method == "POST":
+        form= CreateNewSubsidy(request.POST or None, instance=subsidy)
+        if form.is_valid():
+            print("="*50)
+            print(form.cleaned_data)
```

Comment:
- Be careful! You forgot some prints... Don't worry! This has happened to me more once 😆

Reply:
Upps, sorry. Thanks for the feedback!